serverjs/cubefn.js

```diff
@@ -271,7 +271,7 @@ function CSVtoCards(csvString, carddb) {
         finish: finish || 'Non-foil',
         imgUrl: (imageUrl || null) && imageUrl !== 'undefined' ? imageUrl : null,
         imgBackUrl: (imageBackUrl || null) && imageBackUrl !== 'undefined' ? imageBackUrl : null,
-        tags: tags && tags.length > 0 ? tags.split(',') : [],
+        tags: tags && tags.length > 0 ? tags.split(';').map((t) => t.strip()) : [],
```

Comment:
Is `strip` some kind of undocumented string function I don't know, or is this supposed to be `trim`?